### PR TITLE
Added docstring to `jira_issue` fixture

### PR DESCRIPTION
**Pull request by liiight**
This adds data when running `pytest --fixtures`:
```
-------------------------------------------------------------------------------------------------- fixtures defined from pytest_jira ---------------------------------------------------------------------------------------------------
jira_issue
    /Users/orcarmi/stronghold-py/lib/python3.6/site-packages/pytest_jira.py:450: no docstring available
```

### CODE REVIEW

**Comment by liiight** [comment]
I fucked up my fork from a botched rebase, if the dirty commit history is problematic let me know and ill recreate the branch (although you should be able to squash the commit on the merge and ignore this shit anyway).

File: pytest_jira.py [hunk 1]
```diff
@@ -448,6 +448,10 @@ def pytest_configure(config):
 
 @pytest.fixture
 def jira_issue(request):
+    """
+    Returns a bool representing the state of the issue, or None if no connection could be made.
```
Comment (lukas-bednar) [inline pytest_jira.py]: These lines are just too long, that is why gate fails on pep8.

**Comment by liiight** [comment]
I think this enforcement is way too strict. pep8 is supposed be a style guide, hard limiting to 72 chars per line just too strict, other project relax it to 100 or even 120 chars.

**Comment by lukas-bednar** [comment]
> I think this enforcement is way too strict. pep8 is supposed be a style guide, hard limiting to 72 chars per line just too strict, other project relax it to 100 or even 120 chars.

AFAIR default should be 79. I think going with defaults could not be so strict ;-)
It is just a matter of each project, how it is set there.

Thanks for PR!

**Comment by liiight** [comment]
For docstrings it's 72 AFAIK, but whatever :-)

**Pull request merged**